### ISSUE TITLE
Module not found when using 7.3.0

### ISSUE THREAD

**Issue by ghmj2417** [scttcper/ngx-color#361]
This issue does not occur in 7.2.0.  I am using Angular 12.1

```
Error: Module not found: Error: Package path ./chrome is not exported from package /mydir/node_modules/ngx-color (see exports field in /mydir/node_modules/ngx-color/package.json)
```

It looks like going from 7.2 to 7.3 introduces some breaking changes when using angular 12.  Might be better to move 7.3.0 to 8.0.0 so users using ^7.x.x don't encounter this issue.

**Comment by Timebutt** [comment]
This is the consequence of an error I made in #356 , I have however proposed #360 to fix the issue. I see it just got merged seconds ago, that should fix the problem without the need for a major version bump ;)

**Comment by scttcper** [comment]
fixed in 7.3.1

**Comment by ghmj2417** [comment]
@Timebutt Works for me.  Thank you for fixing it.
